ui/ozone/platform/drm/host/drm_window_host.cc

```diff
@@ -41,12 +41,14 @@ DrmWindowHost::DrmWindowHost(PlatformWindowDelegate* delegate,
 }
 
 DrmWindowHost::~DrmWindowHost() {
+  delegate_->OnAcceleratedWidgetDestroying();
```

Comment:
not needed here, because Drm doesn't take the desktop_window_tree_host_platform path.